.tekton/tasks/task-check-health.yml

```diff
@@ -0,0 +1,36 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
We might want to consider publishing this to the tekton-catalog repo. It's useful to have for everyone.

Reply:
One day